package.json

```diff
@@ -24,6 +24,8 @@
     "lerna": "^3.20.2",
     "nyc": "^14.1.1",
     "puppeteer": "^2.1.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
Why is this added to the root package.json?

Reply:
I was getting errors because the library and sample were using different instances of react even though they were the same version. Installing at the root seemed to fix it but I still want to play around with it to see if there's a better solution

Reply:
Fixed the issues I was having by having the sample import the libraries and react locally. Removed from the root.